query/src/interpreters/interpreter_factory_v2.rs

```diff
@@ -174,6 +172,15 @@ impl InterpreterFactoryV2 {
             }
             Plan::DropStage(s) => DropUserStageInterpreter::try_create(ctx, *s.clone()),
             Plan::RemoveStage(s) => RemoveUserStageInterpreter::try_create(ctx, *s.clone()),
+
+            // Shows
```

Comment:
In the parser, [shows are placed under `EXPLAIN`](https://github.com/andylokandy/databend/blob/ad69ab26630c50811427a1ca87c5947ce7ef5320/common/ast/src/ast/statement.rs#L54). It will be convenient to follow the same order.